pyof/v0x01/common/flow_match.py

```diff
@@ -119,3 +120,27 @@ def __init__(self, wildcards=None, in_port=None, dl_src=None, dl_dst=None,
         self.nw_dst = nw_dst
         self.tp_src = tp_src
         self.tp_dst = tp_dst
+
+        if self.wildcards is None:
+            self.fill_wildcards()
+
+    def fill_wildcards(self):
+        """Fill wildcards considering the attributes of the instance."""
+        self.wildcards = 0
```

Comment:
Why the default wildcards is ``0``? We have decided to go with OFPFW_ALL as default value for the wildcard.

Moreover, It has a 'reverse' logic. So, as it was pointed out on the issue #180, the '*mask*' of this field will 'match' those bits that are zero, not those that are 1.
With that, may be the logic (booleanly speaking) on the following lines on this method may be wrong.
e.g.: If you want a wildcard to match only the dl_src field, you will need:

wildcard = OFPFW_ALL - OFPFW_DL_SRC